app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageComposerViewModel.kt

```diff
@@ -207,29 +208,79 @@ class MessageComposerViewModel @Inject constructor(
         }
     }
 
-    fun sendMessage(sendMessageBundle: SendMessageBundle) {
+    fun sendMessage(messageBundle: MessageBundle) {
```

Comment:
ViewModel has single entry for sending a message, a message  bundle can be of any type and can be extended in the future, the message bundle is being created when the users clicks the send button inside the message composer